torch/onnx/_internal/jit_utils.py

```diff
@@ -42,6 +42,7 @@ class GraphContext:
     original_node: _C.Node
     params_dict: Dict[str, "_C.IValue"]
     env: Dict[_C.Value, _C.Value]
+    new_nodes: List[_C.Node]
```

Comment:
Add field default factory of empty list to make it backward compatible

Reply:
Thanks, I've pushed a fix.